dirty_cat/datasets/tests/test_fetching.py

```diff
@@ -409,19 +129,26 @@ def test_fetch_world_bank_indicator():
         "dataset_columns_count": 2,
     }
 
-    test_data_dir = get_test_data_dir()
-
-    try:
+    with TemporaryDirectory() as temp_dir:
+        temp_dir = Path(temp_dir).absolute()
         try:
             # First, we want to purposefully test FileNotFoundError exceptions.
             with pytest.raises(FileNotFoundError):
-                assert fetch_world_bank_indicator(indicator_id=0)
-                assert fetch_world_bank_indicator(indicator_id=2**32)
+                assert _fetching.fetch_world_bank_indicator(
+                    indicator_id=0, directory=temp_dir
+                )
+                assert _fetching.fetch_world_bank_indicator(
+                    indicator_id=2**32,
+                    directory=temp_dir,
+                )
 
             # Valid call
-            returned_info = fetch_world_bank_indicator(indicator_id=test_dataset["id"])
+            returned_info = _fetching.fetch_world_bank_indicator(
+                indicator_id=test_dataset["id"],
+                directory=temp_dir,
+            )
 
-        except URLError:
+        except (ConnectionError, URLError):
             warnings.warn(
                 "No internet connection or the website is down, test aborted."
             )
```

Comment:
We should maybe use pytest.skip as above?

Reply:
I think it's already the case? I'll remove the warning though!